File: code/1019/1019.cpp
#include <iostream>
using namespace std;

int main() {

  int n, hours, minutes, seconds;

  cin >> n;

  hours = n / 3600;

  minutes = (n - (hours * 3600)) / 60;

  seconds = n - (hours * 3600) - (minutes * 60);

  cout << hours << ":" << minutes << ":" << seconds << endl;

  return 0;
}

File: code/1019/1019.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int duration_in_seconds;
        int hours, minutes, seconds;

        duration_in_seconds = int.Parse(Console.ReadLine());

        hours = duration_in_seconds / 3600;
        duration_in_seconds %= 3600;
        minutes = duration_in_seconds / 60;
        duration_in_seconds %= 60;
        seconds = duration_in_seconds;

        Console.WriteLine("{0}:{1}:{2}", hours, minutes, seconds);
    }
}